src/plugins/intel_cpu/src/cpu_map_scheduling.cpp

```diff
@@ -70,33 +71,37 @@ std::vector<std::vector<int>> apply_hyper_threading(bool& input_ht_hint,
 }
 
 bool get_cpu_pinning(bool& input_value,
-                     const bool input_changed,
-                     const int num_streams,
-                     const Config::LatencyThreadingMode latency_threading_mode,
-                     const std::vector<std::vector<int>>& proc_type_table) {
-    int result_value;
-    int num_sockets = get_default_latency_streams(latency_threading_mode);
-    bool latency = num_streams <= num_sockets && num_streams > 0;
+                     const bool& input_changed,
+                     const std::vector<std::vector<int>>& streams_info_table) {
+    bool result_value;
 
+#if defined(__APPLE__)
+    result_value = false;
+#else
     if (input_changed) {
         result_value = input_value;
```

Comment:
I think we don't support cpu binding in two sockets machine in windows?

Reply:
updated